sources/engine/Xenko.UI/Rendering/UI/UIRenderFeature.Picking.cs

```diff
@@ -19,8 +19,6 @@ public partial class UIRenderFeature
 
         private readonly List<PointerEvent> compactedPointerEvents = new List<PointerEvent>();
 
-        public UIElement UIElementUnderMouseCursor { get; private set; }
```

Comment:
Isn't this a breaking change ? I haven't used the UI much so perhaps its already not recommended ?

Reply:
This is a render feature, I really doubt someone actually have used this property. I didn't know it exist until I was fixing those bugs.
Anyhow, I've returned the property and marked it as obsolete.